db/channel_cache.go

```diff
@@ -173,10 +173,15 @@ func (c *channelCache) wouldBeImmediatelyPruned(change *LogEntry) bool {
 
 }
 
-// Internal helper that prunes a single channel's cache. Caller MUST be holding the lock.
-func (c *channelCache) _pruneCache() {
-	pruned := 0
+func (c *channelCache) pruneCacheLength() {
```

Comment:
Don't need locking version any more - only called during cache add.  Check to see if the log message needs to be preserved elsewhere.